### ISSUE TITLE
[FEATURE REQUEST] Add a way to configure season's values with Enviromine.  (Maybe add more configs aswell)

### ISSUE THREAD

**Issue by HRudyPlayZ** [DarkShadow44/SereneSeasons#18]

**Comment by DarianLStephens** [comment]
I'd like to see this, too. Particularly, the ability to configure season temperature.

**Comment by kotmatross28729** [comment]
If anyone is interested, I added the ability to adjust temperatures with seasons in [my Enviromine fork](https://github.com/kotmatross28729/EnviroMine-continuation)

It is based on the Galaxy Odyssey fork + my fixes & features, so it is the current newest (maintained) fork 

So, I think we can close this one?

**Comment by SilearFlare** [comment]
Yeah I really feel this is by default more of a core feature addition of enviromine than a SS feature
bloke's already working his butt off maintaining the thing as is, enviromine GO is mostly dead (although not remotely as dead as OG enviromine) so it'd make sense to close this and use the fork instead
